Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json

```diff
@@ -4979,6 +4979,19 @@
   "subtitle": "",
   "provider": "Microsoft"
 },
+{
+    "workbookKey": "ThreatIntelligenceWorkbook",
```

Comment:
@samikroy, this "workbookKey" is already used for the threat intelligence workbook in [threat intelligence](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Threat%20Intelligence/Workbooks) solution. Can you please change this?

Reply:
Addressed in PR #7776